DESCRIPTION

```diff
@@ -38,13 +39,26 @@ Suggests:
     readr,
     tidyxl,
     readxl,
-    stringr,
     htmltools,
     rvest,
     selectr,
     DT,
     testthat,
     covr,
-    here
+    here,
+    cowplot,
+    devtools,
+    ggplot2,
+    hrbrthemes,
+    lubridate,
+    magick,
+    plotly,
+    scales,
+    tidyverse
 VignetteBuilder: knitr
 Roxygen: list(markdown = TRUE)
+Remotes: git::https://github.com/ianmoran11/tidyxl.git
```

Comment:
Is this fork still needed?  I'm sorry, I have forgotten what it was for.  If it is required then I will need to merge it into tidyxl before merging this.

Reply:
It shouldn't be. `xlsx_cells_fmt` is now in the unpivotr R folder in the file `tidyxl_xlsx_cells_fmt.R`.

Reply:
Addressed in fa7942dea2c8c091d6c91fdef33831cbbed31915.